definitions/pipelines/bisulfite.cwl

```diff
@@ -30,6 +30,12 @@ inputs:
         type: int
     QCannotation:
         type: File
+#nonmethylation_sites_bed is a variable to determine if user wants to obtain vcf/bed/bedgraph/bigwig for non - methylation sites. Value has to be yes or no
```

Comment:
I would change 
"obtain vcf/bed/bedgraph/bigwig for non - methylation sites"
to
"obtain bed/bigwig files for non-CpG cytosines"